src/test/test-execute.c

```diff
@@ -167,6 +208,8 @@ int main(int argc, char *argv[]) {
                 test_exec_environment,
```

Comment:
Fixed, this branch was based on code before test_exec_passenvironment was added.